plugins/stingray/the_debuginator_plugin/c_api_the_debuginator.cpp

```diff
@@ -292,6 +295,8 @@ void handle_default_input(TheDebuginator* debuginator, unsigned devices) {
 		return;
 	}
 
+	float default_focus_height = 0.3f;
```

Comment:
There's the one in the default config thing. It should really be added to DebuginatorPluginCreateContext and then you can store the app's wanted default value in this function similar to how other config variables are done:

TheDebuginator* create_debuginator(const char* id, DebuginatorPluginCreateContext* context) {